bitnami/etcd/README.md

```diff
@@ -0,0 +1,178 @@
+| `persistence.accessMode`           | PVC Access Mode for etcd volume                                                                            | `ReadWriteOnce`                                                    |
+| `persistence.size`                 | PVC Storage Request for etcd volume                                                                        | `8Gi`                                                              |
+| `persistence.annotations`          | Annotations for the PVC                                                                                    | `{}`                                                               |
+| `nodeSelector`                     | Node labels for pod assignment                                                                             | `{}`                                                               |
+| `tolerations`                      | Toleration labels for pod assignment                                                                       | `[]`                                                               |
+| `resources`                        | CPU/Memory resource requests/limits                                                                        | Memory: `256Mi`, CPU: `250m`                                       |
+| `livenessProbe.enabled`               | would you like a livessProbed to be enabled                                                             |  `true`                                                            |
+| `livenessProbe.initialDelaySeconds`   | Delay before liveness probe is initiated                                                                |  10                                                                |
+| `livenessProbe.periodSeconds`         | How often to perform the probe                                                                          |  10                                                                |
+| `livenessProbe.timeoutSeconds`        | When the probe times out                                                                                |  5                                                                 |
+| `livenessProbe.failureThreshold`      | Minimum consecutive failures for the probe to be considered failed after having succeeded.              |  2                                                                 |
+| `livenessProbe.successThreshold`      | Minimum consecutive successes for the probe to be considered successful after having failed             |  1                                                                 |
+| `readinessProbe.enabled`              | would you like a readinessProbe to be enabled                                                           |  `true`                                                            |
```

Comment:
`would` -> `Would`

Reply:
I would change it to: Turn on and off readiness probe (like we have in the Redis chart)

Reply:
done